scripts/inference/convert_composer_to_hf.py

```diff
@@ -268,7 +268,7 @@ def convert_to_relative_import(
     return '.' + parts[-1]
 
 
-def find_module_file(module_name: str) -> str:
+def find_module_file(module_name: str) -> Optional[str]:
```

Comment:
It looks like I use this function in a way that assumes the output is not `None`, so maybe this should raise an error if `module_file` is `None`.